Yoast/javascript
# Add stemming for Spanish irregular diminutives

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds functionality to stem irregular Spanish diminutives.

## Relevant technical choices:

* I added some specs were we can check paradigms of words for whether they get all stemmed to the same stem.

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test` and check that all unit tests pass.
* Add some other forms from the newly added exception list to the spec and check that those get stemmed correctly.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
